### PR TITLE
Detect `isRepoRoot` on branches with slashes

### LINKED ISSUE #15: `isRepoRoot` doesn't work on branches with slashes
I think I found a way to determine `isRepoRoot`: use the `title`

<table>
<tr>
<th width="50%" align=left>/fregante/github-url-detection 
<td>fregante/github-url-detection: Which GitHub page are you on? Is it an issue? Is it a list? Perfect for your WebExtension or userscript

<tr width="50%" align=left>
<th>/fregante/github-url-detection/tree/npm-test 
<td> fregante/github-url-detection at npm-test
<tr width="50%" align=left>
<th>/fregante/github-url-detection/tree/npm-test/esm
<td> github-url-detection/esm at npm-test · fregante/github-url-detection

</table>

You can see 2 behaviors:

- `isRepoRoot` starts with `getRepoURL`
- `isRepoTree` does not (but **ends** with `getRepoURL`)
	- warning: if the repo is called `fregante/fregante` and the folder is `fregante`, then `title` starts with `fregante/fregante` and ends with `fregante/fregante`

The code might be:

```js
const isRepoRoot = (url) => /^(tree\/[^/]+)?$/.test(getRepoPath(url)!) || (!url && document.title.startsWith(getRepoURL()) && !document.title.endsWith(getRepoURL()))
```
---

Originally discussed at https://github.com/sindresorhus/refined-github/pull/3254#discussion_r442904487

### PR DESCRIPTION
**Pull request by FloEdelmann**
Closes #15.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -462,8 +464,12 @@ const getRepoPath = (url: URL | Location = location): string | undefined => {
 	return undefined;
 };
 
+/** Get the 'user/repo' part from an URL */
+const getRepoURL = (url: URL | Location = location): string => url.pathname.slice(1).split('/', 2).join('/');
+
```
Comment (fregante) [inline index.ts:469]: Can you import the tests for this function from refined GitHub as well?

Comment (FloEdelmann) [reply index.ts:469]: There aren't any (or I missed them) :slightly_frowning_face:

**Comment by fregante** [comment]
Tested, it works!

**Comment by fregante** [comment]
v2 has been published

**Pull request merged**